modules/host/persist.go

```diff
@@ -171,6 +171,44 @@ func (h *Host) load() error {
 	}
 	h.unlockHash = p.UnlockHash
 
+	// COMPAT v1.0.0
+	//
+	// A spelling error in pre-1.0 versions means that, if this is the first
+	// time running after an upgrade, the misspelled field needs to be
+	// transfered over.
+	var compatPersistence struct {
+		FinancialMetrics struct {
+			PotentialStorageRevenue types.Currency `json:"potentialerevenue"`
+		}
+		Settings struct {
+			MinContractPrice          types.Currency `json:"contractprice"`
+			MinDownloadBandwidthPrice types.Currency `json:"minimumdownloadbandwidthprice"`
+			MinStoragePrice           types.Currency `json:"storageprice"`
+			MinUploadBandwidthPrice   types.Currency `json:"minimumuploadbandwidthprice"`
+		}
+	}
```

Comment:
I used git blame to find other fields where compatibility had been accidentally been broken. I'm not sure the best way to be 100% sure that others weren't missed.

I only checked the host fields, not fields from other modules.

There may be some chaos this upgrade.